packages/labs/analyzer/src/lib/javascript/classes.ts

```diff
@@ -76,14 +79,15 @@ export const getClassMembers = (
   const methodMap = new Map<string, ClassMethod>();
   const staticMethodMap = new Map<string, ClassMethod>();
   declaration.members.forEach((node) => {
-    if (ts.isMethodDeclaration(node)) {
+    if (ts.isMethodDeclaration(node) && node.body) {
```

Comment:
This needs a comment like the change in `modules.ts`.

Reply:
This check and all the related `docNode` stuff should be dropped once unique names are no longer required.

Reply:
I kept this because we still want this map to be keyed by member names, but the value is now specifically the implementation signature's `FunctionDeclaration`. Overloads are now discovered by `getFunctionLikeInfo` instead, which reaches them from the implementation signature's node.

Reply:
This has a comment now.